src/Routes.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
-import Main from './pages/Main/Main';
+import SellerListMain from './pages/SellerListMain/SellerListMain';
 
 class Routes extends React.Component {
   render() {
     return (
       <Router>
         <Switch>
-          <Route exact path="/Main" component={Main} />
+          <Route exact path="/sellerListMain" component={SellerListMain} />
```

Comment:
- 라우트 경로는 일반적으로 소문자로 작성하시고, 띄어쓰기는 `-` 로 처리합니다.
```suggestion
          <Route exact path="/seller-list-main" component={SellerListMain} />
```

Reply:
넵 적용했습니다!